.github/ISSUE_TEMPLATE/good_first_issue.md

```diff
@@ -0,0 +1,11 @@
+The purpose of the good first issue label is to highlight which issues are suitable for a new contributor without a deep understanding of the codebase.
```

Comment:
nit: possibly enclose the label name in backticks (in this case, capitalise it like the tag).

Reply:
The tag isn't capitalized? `good first issue` works though